pycompwa/CMakeLists.txt

```diff
@@ -6,7 +6,7 @@ if(${Python3_FOUND})
 
   # Copy to build directory
   configure_file(${CMAKE_CURRENT_SOURCE_DIR}/setup.py.in ${SETUP_PY})
-  add_custom_command(OUTPUT ${OUTPUT}
+  add_custom_target(pycompwa ALL
```

Comment:
This is not the same as the previous implementation. Please revert.

Reply:
Hmm, I had some problems of python files being copied when I made modifications. I will check that again

Reply:
Yeah, I just tried it again. When I make some modifications to some python code (after the build), and I build again, the modifications are not copied to the build directory. Why do you need this `add_custom_command`?